spec/requests/potepan/products_spec.rb

```diff
@@ -4,19 +4,44 @@
 RSpec.describe 'Potepan::Products', type: :request do
   describe 'GET #show' do
     let(:product) { create(:product) }
+    let(:taxon) { create(:taxon) }
+    let(:related_products_list) { create_list(:product, 5, taxons: [taxon]) }
+    let(:related_products_addition) { create(:product, taxons:[taxon], price: "20.20") }
+    let(:related_products) { [related_products_list,related_products_addition].flatten }
 
     before do
+      related_products.each { |related_product| related_product.images << create(:image) }
+      product { |product| product.images << create(:image) }
```

Comment:
なぜブロックを与えているのでしょうか。今の書き方ではブロック内の処理は実行されません。
商品に画像を紐付けたいだけなら単純に下記のようにしましょう。
```suggestion
      product.images << create(:image)
```

Reply:
- [ ] fixed

Reply:
26行目のテスト対応に伴ってコード変更をしました。